README.md

```diff
@@ -181,22 +218,21 @@ recognised:
 
 ## Keyboard Selection
 
-Memtest86+ supports both the legacy keyboard interface (using I/O ports 0x60
-and 0x64) and USB keyboards (using its own USB device drivers). One or the
-other or both can be selected via the boot command line, If not specified on
-the command line, the default is to use both if the system was booted in UEFI
-mode, otherwise to only use the legacy interface.
+Memtest86+ supports both the legacy keyboard interface (using I/O or MMIO
+ports 0x60 and 0x64) and USB keyboards (using its own USB device drivers).
+One or the other or both can be selected via the boot command line, If not
```

Comment:
While at it: let's fix "command line, If" -> "command line. If" :)

Reply:
OK